blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -130,9 +129,6 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             log.debug(String.format("BAD JSON: %s", body));
             log.error(ex.getMessage(), ex);
             DefaultHandler.sendResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
-        } catch (ConnectionException ex) {
```

Comment:
Dont we still have the possibility of getting ConnectionException as long as we are still using Astyanax? I couldnt figure out what actually is throwing this exception before your changes.